src/dynamics.jl

```diff
@@ -9,17 +9,18 @@ function planar_double_integrator(; dt = 0.1, kwargs...)
             0.0 0.0 0.0 1.0
         ],
         B = [
-            dt2 0.0
-            0.0 dt2
-            dt 0.0
-            0.0 dt
+            dt2    0.0
```

Comment:
Isn't there an `m` missing here as well?

Reply:
oof wait how did it not get that change? git...

Reply:
wait no I can't read. yes this is correct as written. states are `position, velocity` so the upper block should not include `mass`. all I did was change the input to `force` from `acceleration`